### PR TITLE
Created dockerfile with ejbca + ejbca-rest

### PR DESCRIPTION
**Pull request by alexv-cpqd**
Created dockerfile with ejbca + ejbca-rest

ejbca-rest certificates path updated to point to EJBCA P12 directory.
EJBCA initialization can take several seconds. ejbca-rest will sleep while EJBCA is not ready.

### CODE REVIEW

File: ejbca-docker/build-ejbca.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export ANT_ARGS="-inputhandler org.apache.tools.ant.input.PropertyFileInputHandler"
+
+# The build uses non-default handler in some cases.  Remove these so we
+# can run fully non-interactively build with PropertyFileInputHandler
+sed -i '/SecureInputHandler/d' bin/cli.xml
+
+$ANT_HOME/bin/ant deploy
+$ANT_HOME/bin/ant install
+
+# 处理一个错误
```
Comment (mmagr) [inline ejbca-docker/build-ejbca.sh]: kudos for the chinese but whaat? :confused: 

Google translate says: 
> \# Handle an error

That might refer to the `-rf` portion of rm. I think it is safe to remove the comment.

File: ejbca-docker/build-ejbca.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+rm -rf $APPSRV_HOME/standalone/configuration/standalone_xml_history/current
+
+# 保存证书
```
Comment (mmagr) [inline ejbca-docker/build-ejbca.sh]: kudos for the chinese but whaat? :confused:

Google translate says:
> \# Save the certificate

I think the translation is fair...

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+/opt/jboss-as-7.1.1.Final/bin/standalone.sh -b 0.0.0.0 &
```
Comment (mmagr) [inline entrypoint.sh]: If I got things right, this should actually be `127.0.0.1` shouldn't it?
It seems to me that only the REST adapter should be able to interact with the CA.

Comment (alexv-cpqd) [reply entrypoint.sh]: True. but at the current development state we still need to access EJBCA web service for manually configure some parameters  (like certificate and endpoint profile creation).

Comment (mmagr) [reply entrypoint.sh]: We're doing that manually?
If so, we must have instructions somewhere - this repo's readme seems a good place to start.

**Pull request merged**